pepper-apis/src/main/resources/db-changes/schema/add_new_event_type_DDP-7415.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="nika_sakandeldize" id="add-dsm-notification-event-type-for-stoolkit-07-02-22">
+        <insert tableName="dsm_notification_event_type">
+            <column name="dsm_notification_event_type_code">STOOL_SENT</column>
+        </insert>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
It looks like this file got caught up in the directory restructure. This should be moved over to `pepper-apis/dss-database/src/main/resources/db-changes/schema/`